hw_2.js

```diff
@@ -0,0 +1,107 @@
+//Task 1
+
+let x = 20;
+let y = 58;
+let z = 42;
+
+x = x+y+z ;
```

Comment:
между операндами и операторами принято ставить пробелы
```suggestion
x = x + y + z ;
```